INFO     root:test_02_usersync.py:146 ==========开始执行用例：usersync_tasks_statistics_01==========
INFO     root:runcase.py:30 -·-·-·-·-·-·-·-·-·-发送请求并接受信息 START-·-·-·-·-·-·-·-·-·-
INFO     root:handleyaml.py:94 Yaml测试用例基础架构检查通过
INFO     root:runcase.py:47 重组后的用例信息：{'name': '查询近7次的同步状态', 'description': '查询近7次的同步状态', 'validate': {'code': 200, 'response': {'statusCode': 0, 'msg': 'success'}}, 'epic': '用户管理', 'feature': '用户同步管理-用户同步', 'story': '查询近几次的同步状态接口', 'base_url': 'https://192.168.148.174:31000', 'request': {'method': 'GET', 'address': '/api/usersync/v1/task/statistics/7', 'headers': {'Content-Type': 'application/json', 'token': 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJhdWQiOiJhcHBsaWNhdGlvbiIsInN1YiI6ImNmMjhmM2M2LTA3YzUtNDRhZS05YWExLTI1Y2Q4MTg4ZTMxZCIsImlzcyI6ImJtc19zZXJ2ZXIiLCJzZXNzaW9uSWQiOiJjZjI4ZjNjNi0wN2M1LTQ0YWUtOWFhMS0yNWNkODE4OGUzMWQiLCJleHAiOjE2ODc1MjAwNTksImlhdCI6MTY4NzM0MDA1OX0.mKGgCUYuJC4NKp9M2EjIwEGx9vLxnPH71YDR71nwWYJzKL2cc_k36u5mcW49jffY2s87f9Urpi2H2Wyrl7cDJA', 'Referer': 'https://192.168.148.174:31000/sub-app-unity/userSync'}}, 'caseid': 'usersync_tasks_statistics_01'}
INFO     root:runcase.py:83 请求地址：https://192.168.148.174:31000/api/usersync/v1/task/statistics/7
INFO     root:runcase.py:84 请求方法：GET
INFO     root:runcase.py:85 请求头：{'Content-Type': 'application/json', 'token': 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJhdWQiOiJhcHBsaWNhdGlvbiIsInN1YiI6ImNmMjhmM2M2LTA3YzUtNDRhZS05YWExLTI1Y2Q4MTg4ZTMxZCIsImlzcyI6ImJtc19zZXJ2ZXIiLCJzZXNzaW9uSWQiOiJjZjI4ZjNjNi0wN2M1LTQ0YWUtOWFhMS0yNWNkODE4OGUzMWQiLCJleHAiOjE2ODc1MjAwNTksImlhdCI6MTY4NzM0MDA1OX0.mKGgCUYuJC4NKp9M2EjIwEGx9vLxnPH71YDR71nwWYJzKL2cc_k36u5mcW49jffY2s87f9Urpi2H2Wyrl7cDJA', 'Referer': 'https://192.168.148.174:31000/sub-app-unity/userSync'}
DEBUG    urllib3.connectionpool:connectionpool.py:1003 Starting new HTTPS connection (1): 192.168.148.174:31000
DEBUG    urllib3.connectionpool:connectionpool.py:456 https://192.168.148.174:31000 "GET /api/usersync/v1/task/statistics/7 HTTP/1.1" 200 705
INFO     root:runcase.py:168 请求接口结果： {'statusCode': 0, 'msg': 'success', 'data': {'status': 'syncing', 'statistics': [{'time': '2023-06-21 17:37:21', 'usergroupSuccCnt': 0, 'userSuccCnt': 0, 'usergroupFailCnt': 0, 'userFailCnt': 0}, {'time': '2023-06-21 17:37:21', 'usergroupSuccCnt': 0, 'userSuccCnt': 0, 'usergroupFailCnt': 0, 'userFailCnt': 0}, {'time': '2023-06-21 17:37:21', 'usergroupSuccCnt': 0, 'userSuccCnt': 0, 'usergroupFailCnt': 0, 'userFailCnt': 0}, {'time': '2023-06-21 17:37:21', 'usergroupSuccCnt': 0, 'userSuccCnt': 0, 'usergroupFailCnt': 0, 'userFailCnt': 0}, {'time': '2023-06-21 17:37:21', 'usergroupSuccCnt': 0, 'userSuccCnt': 0, 'usergroupFailCnt': 0, 'userFailCnt': 0}, {'time': '2023-06-21 17:37:21', 'usergroupSuccCnt': 0, 'userSuccCnt': 0, 'usergroupFailCnt': 0, 'userFailCnt': 0}]}}
INFO     root:checkresult.py:113 code断言通过, 期望状态码:200, 实际状态码:200
INFO     root:checkresult.py:48 返回结果断言通过, 期望返回值:{'statusCode': 0, 'msg': 'success'}, 实际返回值:{'statusCode': 0, 'msg': 'success', 'data': {'status': 'syncing', 'statistics': [{'time': '2023-06-21 17:37:21', 'usergroupSuccCnt': 0, 'userSuccCnt': 0, 'usergroupFailCnt': 0, 'userFailCnt': 0}, {'time': '2023-06-21 17:37:21', 'usergroupSuccCnt': 0, 'userSuccCnt': 0, 'usergroupFailCnt': 0, 'userFailCnt': 0}, {'time': '2023-06-21 17:37:21', 'usergroupSuccCnt': 0, 'userSuccCnt': 0, 'usergroupFailCnt': 0, 'userFailCnt': 0}, {'time': '2023-06-21 17:37:21', 'usergroupSuccCnt': 0, 'userSuccCnt': 0, 'usergroupFailCnt': 0, 'userFailCnt': 0}, {'time': '2023-06-21 17:37:21', 'usergroupSuccCnt': 0, 'userSuccCnt': 0, 'usergroupFailCnt': 0, 'userFailCnt': 0}, {'time': '2023-06-21 17:37:21', 'usergroupSuccCnt': 0, 'userSuccCnt': 0, 'usergroupFailCnt': 0, 'userFailCnt': 0}]}}
INFO     root:runcase.py:63 -·-·-·-·-·-·-·-·-·-发送请求并接受信息 END-·-·-·-·-·-·-·-·-·-